test/integration/edition_edit_test.rb

```diff
@@ -8,31 +8,90 @@ class EditionEditTest < IntegrationTest
     stub_linkables
   end
 
-  should "show document summary and title" do
-    edition = FactoryBot.create(:guide_edition, title: "Edit page title", state: "draft")
-    visit edition_path(edition)
-
-    assert page.has_title?("Edit page title")
-
-    row = find_all(".govuk-summary-list__row")
-    assert row[0].has_content?("Assigned to")
-    assert row[1].has_text?("Content type")
-    assert row[1].has_text?("Guide")
-    assert row[2].has_text?("Edition")
-    assert row[2].has_text?("1")
-    assert row[2].has_text?("Draft")
-  end
+  context "when edition is draft" do
+    setup do
+      edition = FactoryBot.create(:guide_edition, title: "Edit page title", state: "draft")
+      visit edition_path(edition)
+    end
+
+    should "show document summary and title" do
+      assert page.has_title?("Edit page title")
+
+      row = find_all(".govuk-summary-list__row")
+      assert row[0].has_content?("Assigned to")
+      assert row[1].has_text?("Content type")
+      assert row[1].has_text?("Guide")
+      assert row[2].has_text?("Edition")
+      assert row[2].has_text?("1")
+      assert row[2].has_text?("Draft")
+    end
+
+    should "show all the tabs for the edit" do
+      assert page.has_text?("Edit")
+      assert page.has_text?("Tagging")
+      assert page.has_text?("Metadata")
+      assert page.has_text?("History and notes")
+      assert page.has_text?("Admin")
+      assert page.has_text?("Related external links")
+      assert page.has_text?("Unpublish")
+    end
+
+    context "#metadata" do
+      setup do
+        click_link("Metadata")
+      end
+
+      should "show 'Metadata' header and an update button" do
+        within :css, ".gem-c-heading" do
+          assert page.has_text?("Metadata")
+        end
+        assert page.has_button?("Update")
+      end
+
+      should "show slug input box prefilled" do
+        assert page.has_text?("Slug")
+        assert page.has_text?("If you change the slug of a published page, the old slug will automatically redirect to the new one.")
+        assert page.has_field?("artefact[slug]", with: /slug/)
+      end
+
+      should "update and show success message" do
+        fill_in "artefact[slug]", with: "changed-slug"
+        choose("Welsh")
+        click_button("Update")
+
+        assert find(".gem-c-radio input[value='cy']").checked?
+        assert page.has_text?("Metadata has successfully updated")
+        assert page.has_field?("artefact[slug]", with: "changed-slug")
+      end
+    end
+
+    context "when edition is draft" do
```

Comment:
I think this is supposed to be "when edition is published"?

And it should not be within the preceding "when edition is draft" context, but at the same level.